renku/core/commands/dataset.py

```diff
@@ -833,16 +804,10 @@ def tag_dataset():
 
 
 @inject.autoparams()
-def _remove_dataset_tags(name, tags, client: LocalClient, datasets_provenance: DatasetsProvenance):
+def _remove_dataset_tags(name, tags, client: LocalClient):
```

Comment:
Also don't need this if we have a dedicated business logic class that does this.

Reply:
This is still checking if the dataset exists and raises an error. It's also in par with other commands where we have a private method to do some validations and then forward a call to core business logic. So, I'll keep it for now.